tests/ui/src/utilities/accessibility.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from "playwright";
+import { pageFixture } from "../hooks/pageFixture";
+import { AxeBuilder } from "@axe-core/playwright";
+import { createHtmlReport } from "axe-html-reporter";
+
+export default class Accessibility {
+  constructor(page: Page) {
+    pageFixture.page = page;
+  }
+
+  async runAxeCheck(testId: string) {
+    const accessibilityScanResults = await new AxeBuilder({
+      page: pageFixture.page,
+    }).analyze();
+
+    createHtmlReport({
+      results: accessibilityScanResults,
+      options: {
+        projectKey: "uec-cm-" + testId,
+        outputDirPath: "artifacts/" + testId,
+      },
+    });
+
+    return accessibilityScanResults;
+  }
+}
```

Comment:
I thought that there was going to be a error on x numbers of violations check in here

Reply:
it just returns a FAIL on the single test